core/src/banking_stage/consumer.rs

```diff
@@ -719,7 +719,7 @@ mod tests {
                 atomic::{AtomicBool, AtomicU64},
                 RwLock,
             },
-            thread::{Builder, JoinHandle},
```

Comment:
`Builder` was only used by that test to spawn consumer thread - import no longer needed.